### DISCUSSION TITLE
Allow Multi-device feature

### DISCUSSION THREAD

**Discussion by sciamano89** [ente-io/ente#2717] [Enhancements]
I’m an ex-Authy user. I feel like a great feature they had was the ability to allow new devices to be added to the account, as well as ability to lock it on the go so no new devices could be added. Essentially, a form of 2fa that simply required you to have access to at least one device with your account unlocked. 

Without it I’m feeling like I’m really missing a second authenticating factor. The “email verification” option seems counterintuitive because it recommends you don’t store the email’s totp on Ente, but I’m not going to either un-secure my email, make a second 2fa account just for my email, or risk being in a locked out loop.

**Comment by dnred** [comment]
Not exactly what you want but you can enable TOTP in the Photos app. It works when logging in to Auth too. Just don't store that exclusively in Auth.

**Comment by cgmccurdy** [comment]
Came here to suggest this. The setting for multi-device would typically be disabled. Whenever you needed to add a new device, you would go into settings and enable multi-device. After setting up authy on the new device, you would again disable multi-device and logging in on new devices would be disabled. It was a great feature.

**Comment by adamanthil** [comment]
I am very interested in this feature as well. The UX is a huge improvement from Authy, but I am concerned in the unlikely event of a breach that it would be too easy to add a device without my permission.

**Comment by davidjade** [comment]
As another ex-authy user, I too would like to see the ability to restrict adding new devices to my ente auth account. I can manually unlock adding new devices when I need or use recovery if I need to, but in case of a breach another device should not be allowed to be added without my approval. E2EE is great and all but it just takes one slip up if devices can always be added at any time.
